src/helpers/plainAddPlaceholder/readRefTable.test.js

```diff
@@ -3,16 +3,16 @@ import readRefTable from './readRefTable';
 import SignPdfError from '../../SignPdfError';
 
 describe('readRefTable', () => {
-    it('Errors when wrong ref table position is given', () => {
-        const pdf = fs.readFileSync(`${__dirname}/../../../resources/signed.pdf`);
+    it('should be tested', () => {
+        // const pdf = fs.readFileSync(`${__dirname}/../../../resources/signed.pdf`);
```

Comment:
It should indeed be tested.

Reply:
You're right @andres-blanco

Reply:
No. What I meant with "It should indeed be tested" is that the readRefTable function needs some testing. I know it is covered, but coverage != testing. A readRefTable.test.js file is needed that covers at least some edge cases.